Gordon360/Models/Health_Question.cs

```diff
@@ -12,15 +12,11 @@ namespace Gordon360.Models
     using System;
```

Comment:
Transit requests doesn't exist in the DB. Not sure why it's comparing these two files... I guess because they're both classes that are autogenerated and have the same warning header above the using's.